timemachine/fe/single_topology.py

```diff
@@ -1112,6 +1140,8 @@ def setup_intermediate_state(self, lamb) -> VacuumSystem:
             ),
         )
 
+        START_ANGLE_MIN = 0.2
```

Comment:
Point of uncertainty: does this interact with any other terms?

(Interaction with chiral atom restraints looks correct.)